google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerException.java

```diff
@@ -48,4 +55,22 @@ public ErrorCode getErrorCode() {
   enum DoNotConstructDirectly {
     ALLOWED
   }
+
+  /**
+   * Return the retry delay for transaction in milliseconds. Return -1 if this does not specify any
+   * retry delay. In that case, clients should fall back to a locally computed retry delay.
```

Comment:
`In that case, clients should fall back to a locally computed retry delay.`
This statement is only true in the specific case of ABORTED not in all retries. So please remove from here and put it on getRetryDelayMillis